pkg/monitor/master_test.go

```diff
@@ -180,6 +180,8 @@ func TestBalance(t *testing.T) {
 				t.Fatal(len(doc.Monitor.Buckets))
 			}
 
+			t.Fail()
```

Comment:
This is here to demonstrate the reduced test report output. Will not be part of the code.